tests/nc-server/simple_server.c

```diff
@@ -280,9 +280,9 @@ simple_server_shutdown(void * cookie)
 }
 
 /**
- * simple_server(port, nconn_max, shutdown_after, callback, caller_cookie):
- * Run a server which accepts up to ${nconn_max} connections to port ${port}.
- * After receiving a message, call ${callback} and pass it the
+ * simple_server(sockname, nconn_max, shutdown_after, callback, caller_cookie):
```

Comment:
Normally the parameter names here should match the ones used in the function definition too.

s/sockname/addr/ ?